h2o-core/src/main/java/water/api/SchemaMetadata.java

```diff
@@ -423,6 +423,9 @@ private static String getEnumSchemaName(Class<?> type) {
     StringBuffer sb = new StringBuffer(type.getCanonicalName());
     sb.delete(0, sb.indexOf(".")+1);
     sb.setCharAt(0, Character.toUpperCase(sb.charAt(0)));
-    return sb.toString().replace(".", "").replace("$", "");
+    return sb.toString()
```

Comment:
@seb-h2o just curious if we had tests for this functionality

Reply:
it's used at build time: build calls metadata API to generate code